docs/bump.md

```diff
@@ -216,6 +214,15 @@ Some examples
 bump_message = "release $current_version → $new_version [skip-ci]"
 ```
 
+### `annotated_tag`
+
+Whether to create annotated tags or lightweight ones.
+
```

Comment:
If my understand is correct, we can use `cz bump -at` and set it in config to achive it. If so, I'd suggest adding both usages here.